common/models/templates/test/test-trie-traversal.js

```diff
@@ -102,13 +117,15 @@ describe('Trie traversal abstractions', function() {
         assert.isDefined(traversalInner1);
         assert.isArray(child.traversal().entries);
         assert.isEmpty(child.traversal().entries);
+        assert.equal(traversalInner1.p, 1000 / 500500 /* prob of 'the' */);
 
         for(let tChild of traversalInner1.children()) {
           if(tChild.char == 'r') {
             let traversalInner2 = tChild.traversal();
             assert.isDefined(traversalInner2);
             assert.isArray(tChild.traversal().entries);
             assert.isEmpty(tChild.traversal().entries);
+            assert.equal(traversalInner2.p, 607 / 500500 /* prob of 'true', the best 'tr-' entry */);
```

Comment:
Can we have a constant for `PROB_OF_TRUE` as well? Put all three of them together